### ISSUE TITLE
API Browser documentation contains links to Reference Source

### ISSUE THREAD

**Issue by gewarren** [dotnet/dotnet-api-docs#7460]
---
**Issue moved from MicrosoftDocs/feedback#3724**
- Please respond to @cdanek.

---

_From @cdanek on Monday, December 6, 2021 10:39:21 PM_

**Is your feature request related to a problem? Please describe.**

When diving into a specific API call in the .NET ecosystem, I often find myself (for various reasons: curiosity included) diving into the reference source, which was made open in 2014 - a move that is widely considered to be a positive one. That being said, there is a workflow hiccup in this process, which typically involves some fumbling through code on https://referencesource.microsoft.com/ to find the actual external/public method after reading about it on https://docs.microsoft.com/. 

**Describe the solution you'd like**

I would like to see the addition of a new section in the "In This Article" with links to relevant reference source directly. 

I do understand the impact of this request, but I would like to humbly submit it nonetheless. I'll leave the discussion of policy impact to those with a pay grade that exceeds mine.

**Comment by gewarren** [comment]
---
**Issue moved from MicrosoftDocs/feedback#3724**
- Please respond to @welcome[bot].

---

_From @welcome[bot] on Monday, December 6, 2021 10:39:23 PM_

Thank you for creating the issue! One of our team members will get back to you shortly with additional information. If this is a product issue, please close this and contact the particular product's support instead (see https://support.microsoft.com/allproducts for the list of support websites).

**Comment by svick** [comment]
This looks like a duplicate of https://github.com/MicrosoftDocs/feedback/issues/433. (Not sure why this one was moved here.)

@cdanek You may be aware of this, but https://referencesource.microsoft.com/ contains (some of) the source code for the .Net Framework. .Net Core/.Net 5+ is fully open source and you can find that source at https://source.dot.net/ and at https://github.com/dotnet/runtime/.

**Comment by cdanek** [comment]
Thanks, @svick. I did look for a feature request that was similar to this, but I was looking in a different repo (MicrosoftDocs/Feedback). I am aware that the source is available, but I was hoping to streamline the process and get links *directly* to the source from the (more-groomed) public facing documentation at docs.ms.com.

Thank you for your response, and I'll be following the other issue and hoping for its future inclusion! Cheers. I did not open this issue so I cannot close it, but you certainly have my blessing to do so.
